lib/puppet/provider/nssdb/certutil.rb

```diff
@@ -0,0 +1,50 @@
+require 'fileutils'
+
+Puppet::Type.type(:nssdb).provide(:certutil) do
+  commands :certutil => 'certutil'
```

Comment:
I think this is the same
```suggestion
  commands :certutil
```

Reply:
You're right -- I tested it and confirmed it works.

Reply:
Actually I ran the wrong tests. It leads to `undefined method each` error